Lesson006/Task 1 HW06.TheTriangleBuilder/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Task1HW06.TheTriangleBuilder
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int userInput = Convert.ToInt32(Console.ReadLine());
+
+            int value = 1;
+
+            for (int lines = userInput; lines >= 1; lines--)
+            {
+                for (int spaces = userInput - 1; spaces >= lines; spaces--)
+                {
+                    Console.Write(" ");
+                }
+
+                for (int colums = 1; colums <= lines; colums++)
+                {
```

Comment:
Inside an expression inside a loop, it's common to use i, j or k. For example var i = 0; i < array.length; i++

Reply:
Fixed